public/app/content/header-and-footer.css

```diff
@@ -0,0 +1,48 @@
+﻿/* Sticky footer styles
+-------------------------------------------------- */
+html {
+  position: relative;
+  min-height: 100%;
+}
+body {
+  /* Margin bottom by footer height */
+  margin-bottom: 60px;
```

Comment:
This is not good to reduce "html" or "body" container sizes. Margins are set to custom containers usually.